pkg/nfd-master/nfd-master.go

```diff
@@ -214,6 +215,29 @@ func filterFeatureLabels(labels Labels, labelWhiteList *regexp.Regexp) Labels {
 	return labels
 }
 
+// Filter label namespaces and set default namespace to labels without one
+func filterNamespaceLabels(labels Labels, extraLabelNs []string) Labels {
+	allowedLabels := make(Labels)
+L:
+	for labelName, labelValue := range labels {
+		// If there is no namespace in the name
+		if !strings.Contains(labelName, "/") {
+			allowedLabels[labelNs+labelName] = labelValue
```

Comment:
@marquiz Currently I am adding the default namespace here, but I don't like it. I don't know where I should do this to be honest. Maybe another function doing only this? Do you have an idea?

Reply:
I need to call this a day now - I'll get back to this tomorrow

Reply:
I came back for a while 😄 I took a look, and, thought it is easier to write a comment in a form of a patch, see Ethyling/node-feature-discovery#1

Feel free to squash/modify that

Reply:
Looks really better, ty!